### PR TITLE
add iDVx Lab of TongJi University

### PR DESCRIPTION
**Pull request by Tsailgan**
# Contributing to CSrankings

Thanks for contributing to CSrankings! Here are some guidelines to getting your pull request accepted.

_Do not use Excel to edit any .csv files; Excel incorrectly tries to
convert some Google Scholar entries to formulas, corrupting the
database. Use a text editor like emacs or NotePad instead._

**Inclusion criteria**

- [x] Make sure that any faculty you add meet the inclusion
criteria. Eligible faculty include only full-time, tenure-track
faculty members on a given campus who can *solely* advise PhD students in
Computer Science. Faculty not in a CS department or similar who can
advise PhD students in CS can be included regardless of their home
department.

**Updating an affiliation or home page**

- [ ] Update affiliations, home pages, and Google Scholar entries by modifying `csrankings.csv`. For the Google Scholar entry, just use the alphanumeric identifier in the middle of the URL. If none is there, put NOSCHOLARPAGE.

**Adding one or more faculty members (including an entire department)**

- [x] If the department is not yet listed in CSrankings, the entire faculty needs to be added (not just one faculty member).

- [x] Enter each faculty member's [DBLP](http://dblp.org) name, home page, and Google Scholar entry (just the alphanumeric identifier, not the whole URL) by modifying `csrankings.csv`; include disambiguation suffixes like 0001 as needed. If the faculty entry is currently ambiguous, please do not include them. Send mail to the DBLP maintainers (dblp@dagstuhl.de) with a few publications by a particular faculty member; also, open an issue so that when the DBLP database is updated, that faculty member's information can be added.

- [ ] If DBLP has multiple entries for this person, all of them need to be listed. If an alias is not already present in `dblp-aliases.csv`, add it.

- [x] If the institution you are adding is not in the US,
update `country-info.csv`.

other faculty members(Yang Shi and Qing Chen) are submitted in the "issues".

### CODE REVIEW

**Comment by Tsailgan** [comment]
other faculty members (Yang Shi and Qing Chen) are submitted in the "pull request".

File: country-info.csv [hunk 1]
```diff
@@ -210,6 +210,7 @@ Texas A&M at Qatar,asia
 The Hong Kong Polytechnic University,asia
 Tilburg University,europe
 Tokyo Institute of Technology,asia
+TongJi University,asia
```
Comment (emeryberger) [inline country-info.csv]: The "J" should not be capitalized.

**Review by emeryberger — CHANGES_REQUESTED** [review]
In addition to the mismatch between the university name per English usage (see Wikipedia) and the university's own pages (e.g., https://en.tongji.edu.cn/), the *entire* eligible faculty need to be added in a single PR, per the contributions guidelines.

File: country-info.csv (hunk 1 above)
Comment (Tsailgan) [reply country-info.csv]: Thank you for your remind and I have committed a new pull request which fixed the mismatch.

The entire eligible faculty of our lab (https://idvxlab.com/) has been added.

Comment (emeryberger) [reply country-info.csv]: It needs to be the entire faculty of the _CS department_, not just one lab.

**Comment by emeryberger** [comment]
Bump @Tsailgan - if you are planning to add the entire CS faculty, please do so. I'll close the PR if I don't hear back soon.

File: country-info.csv (hunk 1 above)
Comment (Tsailgan) [reply country-info.csv]: > Bump @Tsailgan - if you are planning to add the entire CS faculty, please do so. I'll close the PR if I don't hear back soon.

I will add the entire CS faculty as soon as possible.

**Comment by emeryberger** [comment]
Please resolve conflicts with `csrankings.csv` and I'll merge - thanks.

**Comment by Tsailgan** [comment]
> Please resolve conflicts with `csrankings.csv` and I'll merge - thanks.

Thank you. I will resolve conflicts as soon as possible.

**Comment by Tsailgan** [comment]
> Please resolve conflicts with `csrankings.csv` and I'll merge - thanks.

 Thank you for your patience.I have resolved conflicts with `csrankings.csv` .

**Pull request merged**